cypress/integration/login.js

```diff
@@ -1,27 +1,42 @@
 describe('Sourcebooks login', function() {
+  it('Should display validation for empty user after attempted loggin', function() {
+    cy.visit('/');
+    cy.get('.Select.not-valid').should('not.visible');
+    cy.get('[type="submit"]').click();
+    cy.get('.Select.not-valid').should('be.visible');
+  });
 
-    it('Should display validation for empty user after attempted loggin', function () {
-        
-        cy.visit('/');
-        cy.get('.Select.not-valid').should('not.visible')
-        cy.get('[type="submit"]').click();
-        cy.get('.Select.not-valid').should('be.visible')
-    })
+  it('Should be able to login with role User', function() {
+    cy.visit('/');
+    cy.get('[id="loginForm.userId"]').click({ force: true });
+    cy.get('[aria-label="Demo User"]').click();
+    cy.get('[id="loginForm.role"]').click({ force: true });
+    // cy.get('[aria-label="User"]').click();

```

Comment:
You should not leave commented out code, if code is commented out chances are its not necessary and should be deleted :)

Reply:
Commented line was deleted :)